src/main/java/com/connexta/store/controllers/StoreController.java

```diff
@@ -128,9 +135,47 @@ public StoreController(@NotNull final StoreService storeService) {
     return ResponseEntity.ok().build();
   }
 
-  @Override
+  @ApiOperation(
+      value = "Get a Product.",
+      nickname = "retrieveProduct",
+      notes = "Clients send a Product ID to retrieve the Product as a file.",
+      response = Resource.class,
+      tags = {"store"})
+  @ApiResponses(
+      value = {
+        @ApiResponse(code = 200, message = "Get Product", response = Resource.class),
+        @ApiResponse(
+            code = 401,
+            message = "The client could not be authenticated. ",
+            response = ErrorMessage.class),
+        @ApiResponse(
+            code = 400,
+            message =
+                "The client message could not be understood by the server due to invalid format or syntax. ",
+            response = ErrorMessage.class),
+        @ApiResponse(
+            code = 403,
+            message = "The client does not have permission. ",
+            response = ErrorMessage.class),
+        @ApiResponse(
+            code = 501,
+            message = "The requested API version is not supported and therefore not implemented. ",
+            response = ErrorMessage.class),
+        @ApiResponse(
+            code = 200,
+            message = "Any other possible errors not currently known. ",
+            response = ErrorMessage.class)
```

Comment:
❗️ We don't want to return the a 200 and an error message for unknown cases.

Reply:
Done. I also pushed a commit to https://github.com/connexta/ion-store-api/pull/8 to remove that response, just like we did with the ingest-api.